app/src/main/java/com/codepath/travel/models/StoryPlace.java

```diff
@@ -43,6 +48,15 @@ public StoryPlace(Trip trip, Place place) {
         setLatitude(latlng.latitude);
         setLongitude(latlng.longitude);
         setPlaceTypes(place.getPlaceTypes());
+        setRating(place.getRating());
```

Comment:
This rating place wasn't meant to house the rating from the Places API. It's supposed to be for internal app ratings (i.e. the user's "favorite" rating, for sorting the collage view).